src/azure-cli/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -425,3 +426,45 @@ def _server_list_custom_func(client, resource_group_name=None):
     if resource_group_name:
         return client.list_by_resource_group(resource_group_name)
     return client.list()
+
+
+# region private_endpoint
+def _update_private_endpoint_connection_status(cmd, client, resource_group_name, server_name,
+                                               private_endpoint_connection_name, is_approved=True, description=None,
+                                               connection_id=None):  # pylint: disable=unused-argument
+    PrivateEndpointServiceConnectionStatus = cmd.get_models('PrivateEndpointServiceConnectionStatus',
+                                                            resource_type=ResourceType.MGMT_RDBMS)
+
+    private_endpoint_connection = client.get(resource_group_name=resource_group_name, server_name=server_name,
+                                             private_endpoint_connection_name=private_endpoint_connection_name)
+
+    new_status = PrivateEndpointServiceConnectionStatus.approved \
+        if is_approved else PrivateEndpointServiceConnectionStatus.rejected
+    private_endpoint_connection.private_link_service_connection_state.status = new_status
+    private_endpoint_connection.private_link_service_connection_state.description = description
+
+    return client.put(resource_group_name=resource_group_name,
+                      server_name=server_name,
+                      private_endpoint_connection_name=private_endpoint_connection_name,
+                      properties=private_endpoint_connection)
+
+
+def approve_private_endpoint_connection(cmd, client, resource_group_name, server_name, private_endpoint_connection_name,
+                                        approval_description=None, connection_id=None):
```

Comment:
hello. Based on our discussion,  we prefer having same argument called `--description` in both `approve` and `reject` command

Reply:
done